exo/src/daemon.rs

```diff
@@ -89,10 +89,15 @@ pub async fn cmd_daemon(ctx: &Context) -> anyhow::Result<()> {
                     }
                 };
 
-                let entities_index_config = EntityIndexConfig::default();
+                let entities_index_config: EntityIndexConfig = config
+                    .clone()
+                    .store
+                    .and_then(|s| s.index)
+                    .unwrap_or_default()
```

Comment:
@appaquet Maybe an `expect` would make more sense here ? Wasn't sure.

Reply:
In fact, what we want here is `EntityIndexConfig::default`. At the point you call `unwrap_or_default`, you still have the protobuf message. This means that `unwrap_or_default` actually returns the empty protobuf that is then converted to `EntityIndexConfig` with `.into()`. What we want is probably more the default value of `EntityIndexConfig` if the proto is not defined. It probably doesn't change much since I'd expect the `From<...Proto...>` does the same work if all fields of the proto are empty?

This is what I came up with:
```
let entities_index_config: EntityIndexConfig = config
    .store
    .as_ref()
    .and_then(|s| s.index.as_ref())
    .map(|e| EntityIndexConfig::from(e.clone()))
    .unwrap_or_default();
```

Note that I removed the `clone()` on the store config since we only one the index config to be cloned here. You probably fought the borrow checker because you were yanking the value out of an optional by mapping it. By calling `.as_ref()` on the optional field, you convert it to a `Option<&T>`, so you're not moving out the value.